src/org/jetbrains/plugins/scala/lang/refactoring/move/MoveScalaClassHandler.java

```diff
@@ -16,6 +16,7 @@
 import org.jetbrains.plugins.scala.lang.psi.api.ScalaFile;
 import org.jetbrains.plugins.scala.lang.psi.api.toplevel.ScNamedElement;
 import org.jetbrains.plugins.scala.settings.ScalaApplicationSettings;
```

Comment:
Sorry about that, I updated the commit to remove the unused ScalaFormatter.  The `Move` action leaves empty line in the source file. 
I considered removing them by reformatting the source file after the move. However a quick check showed that leaving empty lines for removed elements was the current behaviour. I stopped trying to add reformat but forgot to remove the import.